source/_integrations/onkyo.markdown

```diff
@@ -25,15 +25,21 @@ media_player:
     name: receiver
     sources:
       pc: 'HTPC'
+    zones:
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.